src/main/java/robotbuilder/graph/CommandCellEditorDialog.java

```diff
@@ -68,7 +68,7 @@ public CommandCellEditorDialog(CommandGraph graph, CommandCell command) {
 
     public void save() {
         // Save the parameters
-        Vector<Vector<Object>> dataVector = parameterTable.getModel().getDataVector();
+        Vector<Vector> dataVector = parameterTable.getModel().getDataVector();
```

Comment:
Erm, no, you should use `Vector<Vector<*>>` if you need to.

Or, if you want the compiler to infer the type with JDK 11 just use `val` for the type.

Reply:
`parameterTable.getModel().getDataVector();` actually returns a Vector of Vector. The internal vector has no return type in Java 11. And I would like to keep Java 8 building I think, so no `var`.